### PR TITLE
Blog - Developer Tools for Kubernetes

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
@jalletto please look into it

**Comment by msali123** [comment]
@jalletto any update??

**Comment by msali123** [comment]
@jalletto pushed again

**Comment by jalletto** [comment]
Preview URL: https://62f5e5b093363671a3a980a2--ephemeral-narwhal-3ac3b2.netlify.app/

File: blog/_posts/2022-08-20-devetools-for-k8s.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+There are other tool options in the market like [K9s](https://k9scli.io/), however, that's a tool designed for the terminal, and it can only show you one cluster at a time. In addition, because it runs in the terminal, it's limited by the design choices of the terminal, where Lens is able to be a more fully-fledged application.
+
+When you first get started with Lens, it can be a bit confusing because you get overloaded with information about your cluster. However, once you start using it and figure out what you need to look at, it can quickly become an integrated part of your workflow.
+
+### Gitpod
+
+<div class="wide">
+
+![GitPod]({{site.images}}{{page.slug}}/CGnQ1sT.png)\
```
Comment (jalletto) [inline blog/_posts/2022-08-20-devetools-for-k8s.md:49]: Can we switch this image out for this one https://tecnoticias.net/wp-content/uploads/2021/04/gitpod-open-source-developer-platform-vs-code-devxconf-1024x494-768x371.png

**Review by jalletto — COMMENTED** [review]
This looks good. Just one change to make.

File: blog/_posts/2022-08-20-devetools-for-k8s.md (hunk 1 above)
Comment (msali123) [reply blog/_posts/2022-08-20-devetools-for-k8s.md]: Done

**Comment by msali123** [comment]
@jalletto  please check if it is a Jekyll Build issue.

**Comment by msali123** [comment]
@jalletto any update?

**Comment by jalletto** [comment]
@msali123 I'm not sure what the issue was with the build but I reran it and it seemed to work. 

The image you added is in the wrong folder so it's not showing up on the page 
https://62ffe4aa44ba2e2b01d4fd60--ephemeral-narwhal-3ac3b2.netlify.app/blog/devetools-for-k8s/

**Comment by msali123** [comment]
https://630075fb56d6da125980b97f--ephemeral-narwhal-3ac3b2.netlify.app

@jalletto done

**Review by jalletto — APPROVED** [review]

**Pull request merged**